.github/workflows/build-package-docs.yaml

```diff
@@ -0,0 +1,95 @@
+name: Ansible package docs build
+on:
+  workflow_dispatch:
+    inputs:
+      github_fork:
+        description: 'GitHub Fork'
+        required: true
+        default: 'ansible'
+      github_branch:
+        description: 'GitHub Branch'
+        required: true
+        default: 'devel'
+      language:
+        type: choice
+        description: 'Language'
+        required: true
+        default: 'english'
+        options:
+        - 'english'
+        - 'japanese'
+      package_version:
+        type: choice
+        description: 'Ansible Version'
+        required: true
+        default: 'devel'
+        options:
+        - 'devel'
+        - '10'
+        - '9'
+        - '8'
+        - '7'
+      latest_symlink:
+        type: boolean
+        description: 'Add latest symlink'
+        required: true
+
+jobs:
+  build-package-docs:
+    runs-on: ubuntu-latest
```

Comment:
I see that you quote some strings but not others. Why? It seems hugely inconsistent.

Reply:
I work in mysterious ways...

Reply:
Not to be too much of a smarty pants, this is a valid point and I appreciate it. This should be fixed but maybe one or two stragglers that I'll also get.

Reply:
Totally. I think this is consistent now, except for the version list in the inputs, maybe, which could all be unquoted (the problem is only with floats like `3.10` where the *YAML* parser would treat that as a number `3.1` but not with integers like `10`).